lib/css/components/post-summary.less

```diff
@@ -309,38 +305,94 @@
 
 .s-post-summary__watched {
     background-color: var(--yellow-050);
-}
 
-.s-post-summary__deleted {
-    background-color: var(--red-050);
+    .s-post-summary--content-title > a {
+        color: var(--blue-700);
 
-    .is-deleted {
-        color: @white;
-        background-color: var(--red-500);
+        &:hover,
+        &:active,
+        &:visited {
+            color: var(--blue-700);
+        }
+    }
+
+    .s-post-summary--stats {
+        color: var(--black-500);
+    }
+
+    .s-user-card {
+        .s-user-card--link {
+            color: var(--blue-700);
+        }
+
+        .s-user-card--rep,
+        .s-user-card--time {
+            color: var(--black-600);
+        }
     }
 }
 
-.s-post-summary__ignored,
 .s-post-summary__deleted {
-    .s-post-summary--content > *:not(.s-post-summary--content-menu-button):not(.s-post-summary--meta) {
-        opacity: calc(var(--_o-disabled-static) * 1.2); // 0.5 * 1.2 = 0.6
+    background-color: var(--red-025);
+
+    .s-badge__filled {
+        color: var(--white);
+
+        .dark-mode({
+            background-color: var(--red-800)
+        });
     }
-    // TODO DEPRECATED remove `:not(.is-deleted)`
-    .s-post-summary--stats-item:not(.s-badge):not(.is-deleted) {
-        opacity: calc(var(--_o-disabled-static) * 1.2); // 0.5 * 1.2 = 0.6
-        filter: grayscale(100%);
+
+    .is-deleted {
+        color: var(--white);
+        background-color: var(--red-600);
     }
 
-    .s-post-summary--content-title a {
-        color: var(--black-600);
+    .s-post-summary--meta-tags > a {
+        background-color: var(--black-075);
+    }
+}
 
-        &:hover,
-        &:active {
+.s-post-summary__ignored {
+
+    .s-post-summary--stats {
+        color: var(--black-500);
+
+        .s-post-summary--stats-item {
+
+            &.has-answers {
+                color: var(--black-600);
+                border: 1px solid var(--black-075);
+                background-color: var(--black-075);
+
+                .has-accepted-answer {
+                    color: var(--black-600);
+                    background-color: var(--black-075);
+                    border-color: var(--black-075);
+                }
+            }
+        }
+
+        .s-post-summary--stats-item__emphasized {
             color: var(--black-500);
         }
+    }
+
+    .s-post-summary--meta-tags > a {
+        background-color: var(--black-050);
+    }
```

Comment:
The interactive states of tags within ignored post summaries are still blue. Is this as expected?

Hovered tag screenshot:
![image](https://user-images.githubusercontent.com/647177/187980849-71fa522f-b76a-4a5c-8727-94c460b9a3c5.png)

Reply:
Thanks for catching this! It should not be blue, it should be one darker at background black-075.